draft-duke-quic-load-balancers.md

```diff
@@ -192,6 +192,48 @@ two bits of the connection ID to multiplex incoming SCIDs over these schemes.
 
 ### Load Balancer Actions
 
+The load balancer selects a number of bytes of the server connection ID
```

Comment:
bytes or bits?

Reply:
I intended it to be bytes. The reason we used bits in the Obfuscated CID is that it was too easy to determine the routing bytes without more permutations.